pages/curriculum/[lesson].tsx

```diff
@@ -6,33 +6,51 @@ import LessonTitleCard from '../../components/LessonTitleCard'
 import AlertsDisplay from '../../components/AlertsDisplay'
 import ChallengeMaterial from '../../components/ChallengeMaterial'
 import GET_APP from '../../graphql/queries/getApp'
-import { Lesson, LessonStatus } from '../../@types/lesson'
-import { UserSubmission } from '../../@types/challenge'
-import { AppData } from '../../@types/app'
+import {
+  Challenge,
+  Lesson,
+  UserLesson,
+  Submission,
+  GetAppQuery
+} from '../../graphql/index'
 import withQueryLoader, {
   QueryDataProps
 } from '../../containers/withQueryLoader'
 import _ from 'lodash'
 
-const Challenges: React.FC<QueryDataProps<AppData>> = ({ queryData }) => {
+const Challenges: React.FC<QueryDataProps<GetAppQuery>> = ({ queryData }) => {
   const { lessons, session, alerts } = queryData
   const router = useRouter()
-  const currentlessonId = router.query.lesson as string
+  const currentlessonId = Number(router.query.lesson) as number
```

Comment:
This is like if you said `Take this number as a number`, pretty redundant right? The compiler is smart enough to figure out that it's a number haha.

```suggestion
  const currentlessonId = Number(router.query.lesson)
```